### DISCUSSION TITLE
vim normal mode in terminal normal fzf prompt?

### DISCUSSION THREAD

**Discussion by semi710** [junegunn/fzf#3294] [Q&A]
how can i enable vim normal mode in fzf prompt in terminal?

**Comment by thjbdvlt** [comment]
i think you cannot just enable normal mode vim, but you can tinker something with bindings, e.g. `bind` `j` and `k` to `down` and `up`, and `unbind` both when fzf start (`load`), then, when `esc` is pressed, `rebind` them:

```
fzf \
    --bind="j:down" \
    --bind="k:up" \
    --bind="i:enable-search+unbind(j)+unbind(k)+unbind(i)"
    --bind="load:unbind(j)+unbind(k)+unbind(i)" \
    --bind="esc:disable-search+rebind(j)+rebind(k)+unbind(i)" \
```

of course, it is possible to write a pretty little script that would build that command for you, as it is very repetitive.
